scripts/cluster/start.sh

```diff
@@ -68,7 +68,12 @@ if ! gsutil du "gs://${BUCKET_NAME}/"; then
   gsutil mb -l eu "gs://${BUCKET_NAME}/"
 fi
 
-gsutil acl ch -u "${GSA_EMAIL}:O" "gs://${BUCKET_NAME}"
+if gsutil uniformbucketlevelaccess get "gs://${BUCKET_NAME}" | grep -q "Enabled: True"; then
```

Comment:
I had issues with this before, did you test in both cases this works? I couldn't get uniform bucket acls to work with gcsfuse, but maybe I did something wrong.

Reply:
I checked it with uniformbucketlevelaccess enabled and can confirm that it's working.